scripts/build/docs_part_1.sh

```diff
@@ -1,11 +1,26 @@
+#comment this below line in local if nvm already installed
```

Comment:
this part of code from line 1-14, can they be in node_install.sh? or they won't work if they are in node_install.sh?

Reply:
if we put in different script its not inheriting

Reply:
could you also check in your local build with this upgrade node, the file under src/main/content/antora_ui/build/ui-bundle.zip is generated? thanks

Reply:
yes its ui-bundle.zip is generating